### PR TITLE
Make session start time required

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
Looks like this wasn't required on the metadata schema side

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
Should we also include things like `identifier` here even though they're most often easy to generate by default?

**Review by bendichter — DISMISSED** [review]

**Comment by CodyCBakerPhD** [comment]
Oh, should we also make `session_id` required? Since that's what's used (in addition to other fallback fields) to name the files on DANDI

File: src/neuroconv/schemas/base_metadata_schema.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "NWBFile": {
       "type": "object",
       "additionalProperties": false,
+      "required": ["session_start_time"]
```
Comment (bendichter) [inline src/neuroconv/schemas/base_metadata_schema.json:13]:
```suggestion
      "required": ["session_start_time"],
```

**Comment by CodyCBakerPhD** [comment]
@bendichter ping

**Review by bendichter — APPROVED** [review]

**Pull request merged**